Analyzer/test/condor/hadder.py

```diff
@@ -165,7 +165,6 @@ def main():
                                 "2016preVFP_AllTT",     "2016postVFP_AllTT",     "2017_AllTT",     "2018_AllTT",
                                 "2016preVFP_RPV",       "2016postVFP_RPV",       "2017_RPV",       "2018_RPV",
                                 "2016preVFP_StealthSYY","2016postVFP_StealthSYY","2017_StealthSYY","2018_StealthSYY",
-                                "2016preVFP_TTX",       "2016postVFP_TTX",       "2017_TTX",       "2018_TTX"
```

Comment:
Yeah, this is fine. I figured it was not too much of a burden to the user to have O(10) additional hadded files in their final output directory 😛

Reply:
Right. I just wanted to have a TTX.root file and with this line you don't get that by default.